Base/res/html/misc/new-tab.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>New Tab</title>
+    <style>
+        #container {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+
+        #main {
+            text-align: center;
+            display: block;
+            width: 100%;
+            max-width: 400px;
+        }
+
+        #image {
+            image-rendering: pixelated;
+        }
+
+        #input {
+            width: 100%;
+            padding: 5px;
+        }
+
+        #search-buttons {
+            display: flex;
+            justify-content: space-between;
+        }
+
+        button:hover {
+            cursor: pointer;
+        }
+    </style>
+</head>
+<body>
+    <div id="container">
+        <div id="main">
+            <br>
+            <img src="/res/icons/32x32/app-browser.png" width="64" height="64" id="image"><br><br>
```

Comment:
Adding an id as generic as `image` is pointless, we can style `img` instead - or use a specific one, like `browser-icon`.

Reply:
My webdev skills are a bit dated but the general idea is to search the DOM tree quicker, I can't speak for Browser but it was a general recommendation a while back, matching `id` stops a search at first match while `img` keeps searching the DOM for matches even after finding one. So that was the general idea behind it. Is the recommendation still current? No idea! Does it help in Browser? No idea! But I figured it couldn't hurt ;-)

Reply:
That's true when using getElementById vs querySelectorAll. It's not true when using querySelector :^)

In the case of CSS, it won't matter on such a small page.